src/astro/sql/operators/agnostic_save_file.py

```diff
@@ -197,9 +196,7 @@ def save_file(
     :type task_id: str
     """
 
-    task_id = (
-        task_id if task_id is not None else get_task_id("save_file", output_file_path)
-    )
+    task_id = task_id if task_id is not None else get_task_id("save_file", "")
```

Comment:
This will return `'save_file_'`, is this intentional?

Reply:
@kaxil, this isn't intentional. We are trying to prevent conflicting names in the DAG, in case there are multiple usages of the same operator. 
I guess we can use `task_id =  task_id.rstrip("_")`

Reply:
`task_id = task_id.rstrip("_") ` was resulting in duplicate task id. Have updated it to use path for task_id creation.